### PR TITLE
Add sponsor group table

### PR DESCRIPTION
**Pull request by kmbibic**
@OttawaU-RSSP/team
Add table to hold info on the sponsor group (from initial web form)

### CODE REVIEW

File: db/migrate/20151015190030_create_sponsor_group.rb [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+      t.text :address
+      t.string :phone
+      t.string :email
+      t.string :citizenship_status
+      t.string :refugee_connection
+      t.integer :group_size
+      t.boolean :SAH_connection
+      t.boolean :interpreter_needed
+      t.boolean :sufficient_resources
+      t.boolean :criminal_record
```
Comment (kwestein) [inline db/migrate/20151015190030_create_sponsor_group.rb:21]: I would add t.timestamps

File: db/migrate/20151015190030_create_sponsor_group.rb [hunk 2]
```diff
@@ -0,0 +1,17 @@
+class CreateSponsorGroup < ActiveRecord::Migration
+  def change
+    create_table :sponsor_groups do |t|
+      t.string :name
+      t.text :address
```
Comment (kwestein) [inline db/migrate/20151015190030_create_sponsor_group.rb]: We'll probably need to break this out into fields
ie: address1, address2, city, postal code, ..

File: db/migrate/20151015190030_create_sponsor_group.rb [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+      t.string :phone
+      t.string :email
+      t.string :citizenship_status
+      t.string :refugee_connection
+      t.integer :group_size
+      t.boolean :SAH_connection
```
Comment (stevelounsbury) [inline db/migrate/20151015190030_create_sponsor_group.rb]: `SAH` should be lowercase

File: db/migrate/20151015190030_create_sponsor_group.rb [hunk 4]
```diff
@@ -0,0 +1,17 @@
+class CreateSponsorGroup < ActiveRecord::Migration
+  def change
+    create_table :sponsor_groups do |t|
+      t.string :name
+      t.text :address
+      t.string :phone
+      t.string :email
+      t.string :citizenship_status
+      t.string :refugee_connection
```
Comment (kwestein) [inline db/migrate/20151015190030_create_sponsor_group.rb]: We'll have to figure out what we want to do about this. ie I think it needs a boolean that says if they have a connection + the relationship + if they're outside country of origin.

**Comment by kwestein** [comment]
:cool: you'll want to create an actual model file too

File: db/migrate/20151015190030_create_sponsor_group.rb [hunk 5]
```diff
@@ -0,0 +1,17 @@
+class CreateSponsorGroup < ActiveRecord::Migration
+  def change
+    create_table :sponsor_groups do |t|
+      t.string :name
+      t.text :address
+      t.string :phone
+      t.string :email
```
Comment (stevelounsbury) [inline db/migrate/20151015190030_create_sponsor_group.rb]: Add a `null: false` to the phone, email and name fields

**Comment by kwestein** [comment]
:cool: looks much better. We can always add more as we need it.
We'll want to gitignore the ._ files though

**Comment by kwestein** [comment]
ie add this to your .gitignore file:
# ignore osx's ._ files
._*

**Comment by stevelounsbury** [comment]
:shipit:

**Pull request merged**